src/lib/jsonParse/jsonParse.cpp

```diff
@@ -378,7 +378,7 @@ static std::string jsonParse
 
   boost::property_tree::ptree subtree = (boost::property_tree::ptree) v.second;
   int                         noOfChildren = subtree.size();
-  if ((isCompoundPath(path.c_str()) == true) && (nodeValue == "") && (noOfChildren != 0))
+  if ((isCompoundPath(path.c_str()) == true) && (nodeValue == "") && (noOfChildren != 0) && treated == true)
```

Comment:
According to [S3 code style rule](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#s3-parenthesis-in-part-conditions) you should use:

```
(treated == true)
```

Reply:
Fixed in 54d6633b408a96d43d2efabd1ae5198d87f75bdc